src/go/wsl-helper/pkg/dockerproxy/serve.go

```diff
@@ -48,32 +64,313 @@ func serve(endpoint string, dialer func() (net.Conn, error)) error {
 		}
 	}()
 
-	for {
-		clientConn, err := listener.Accept()
-		if err != nil {
-			if errors.Is(err, errListenerClosed) {
-				// If the connection is already closed, just return
-				return nil
-			}
-			fmt.Printf("error accepting: %s\n", err)
-			continue
-		}
+	logWriter := logrus.StandardLogger().Writer()
+	defer logWriter.Close()
+	munger := newRequestMunger()
+	proxy := &httputil.ReverseProxy{
+		Director: func(req *http.Request) {
+			logrus.WithField("request", req).
+				WithField("headers", req.Header).
+				WithField("url", req.URL).
+				Debug("got proxy request")
+			// The incoming URL is relative (to the root of the server; we need to
```

Comment:
unclosed paren

Reply:
Fixed in f97b5985db03b47b1db89e6e0a64dab35389ebc3.